src/components/ReanimatedSwipeable.tsx

```diff
@@ -544,23 +580,36 @@ const Swipeable = forwardRef<SwipeableMethods, SwipeableProps>(
       swipeableMethods,
     ]);
 
-    const animatedStyle = useAnimatedStyle(() => ({
-      transform: [{ translateX: appliedTranslation.value }],
-      pointerEvents: rowState.value === 0 ? 'auto' : 'box-only',
-    }));
+    const animatedStyle = useAnimatedStyle(
+      () => ({
+        transform: [{ translateX: appliedTranslation.value }],
+        pointerEvents: rowState.value === 0 ? 'auto' : 'box-only',
+      }),
+      [appliedTranslation, rowState]
+    );
 
     const composedGesture = Gesture.Race(panGesture, tapGesture);
+
+    const containerStyle = props.containerStyle;
+    const childrenContainerStyle = props.childrenContainerStyle;
+    const remainingProps = { ...props };
+
     return (
       <Animated.View
         onLayout={onRowLayout}
-        style={[styles.container, props.containerStyle]}>
+        style={[styles.container, containerStyle]}>
         {leftElement}
         {rightElement}
         <GestureDetector
           gesture={composedGesture}
           touchAction="pan-y"
-          {...props}>
-          <Animated.View style={[animatedStyle, props.childrenContainerStyle]}>
+          {...remainingProps}>
```

Comment:
This way you cannot 'swipe back' the swipeable. I think that pan should be attached to the outer `AnimatedView`, and tap to the inner one.